### ISSUE TITLE
OnlyV not working

### ISSUE THREAD

**Issue by Kulotski66** [AntonyCorbett/OnlyV#33]
Hi all,

I have installed OnlyV on my other laptop with Windows10 but when I start OnlyV its showing the OnlyV box but unable to drag & drop the epub file but on my other laptop running windows7 its working fine.

Thanks in advance if you can give me the solution.

**Comment by AntonyCorbett** [comment]
@Kulotski66 Please run OnlyV as Administrator to see if it's a problem with permissions.

**Comment by Kulotski66** [comment]
Thanks for the fast response, I have tried to run as administrator but still same issue, but during installation I have encountered an error regarding adding shortcut in the desktop "IPersistFile::Save Failed; code 0x80070002. The system cannot find the file specified." then when I reinstall again and un-tick the create shortcut this error disappear but the app still not working and only show the below screen. Thanks
![onlyv](https://user-images.githubusercontent.com/52312354/60328390-42bc5780-999f-11e9-87c6-f8dc04d591d8.jpg)

**Comment by AntonyCorbett** [comment]
@Kulotski66 Thanks for the image - that's certainly not what it should look like. Can you confirm you are using the latest version (1.2.0.10)? Also please would you try downloading and re-running the installer again in case it was corrupted? FInally, if it's still not working please would you send me the log file (in the Documents\OnlyV\Logs folder?

**Comment by Kulotski66** [comment]
@AntonyCorbett got it working, what I did as per your instruction download again then install but still same issue then I have uninstall again then install back but now what I is install in the directory "Program Files" before it was installing under the directory "Program Files (x86). 

Before I re-install I perform the below.
 I was searching for the logs but the directory "Documents\OnlyV" do not exist, so what I did again is to create the directories "Documents\OnlyV\Logs", "Documents\OnlyV\SourceEpubFiles\" and "Documents\OnlyV\ThemeFiles".
Then I copied the .epub file on the directory "Documents\OnlyV\SourceEpubFiles\".

Then I install OnlyV and no errors such as "IPersistFile::Save Failed; code 0x80070002. The system cannot find the file specified." and shortcut has been created and OnlyV works. YES.

Thank you for your support
